src/components/App.js

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import Display from './Display';
+import ButtonPanel from './ButtonPanel';
+
+function App() {
+  return (
+    <div>
```

Comment:
[Optional]
- Kindly wrap these components with React.Fragment short syntax `<> </>`. To avoid creating unnecessary or extra nodes to the DOM